go/teams/teams.go

```diff
@@ -770,7 +770,7 @@ func (t *Team) changeMembershipSection(ctx context.Context, req keybase1.TeamCha
 	section.PerTeamKey = perTeamKeySection
 
 	section.CompletedInvites = req.CompletedInvites
-
+	section.Implicit = t.chain().IsImplicit()
```

Comment:
is it ok for it to be here? The only valid sigs for implicit teams are post root and change membership; but I guess it could also be somewhere else so it would add this parameter for every SCTeamSection